problems/isMultipleOfFive.js

```diff
@@ -13,8 +13,14 @@
 * returns: false (because the string "ten" is not a number)
 */
 
-function isMultipleOfFive() {
-
-}
+function isMultipleOfFive(n) {
+    if(n * 5) {
+        return true;
+    } else if (!num) {
+        return n;
+    } else {
+        return false
+    }
```

Comment:
In case we could see if the reminder of 5 and the number is 0. If so then it is a multiple of 5